test/kernel_acl.js

```diff
@@ -319,6 +321,88 @@ contract('Kernel ACL', accounts => {
                     })
                 })
             })
+
+            context('> burning permission manager', () => {
+                const MOCK_ROLE = keccak256("MOCK_ROLE")
+                let BURN_ENTITY
+
+                before(async () => {
+                    BURN_ENTITY = await acl.BURN_ENTITY()
+                })
+
+                it('burns existing permission', async () => {
+                    // create permission
+                    await acl.createPermission(granted, kernelAddr, MOCK_ROLE, granted, { from: permissionsRoot })
+
+                    // burn it
+                    const receipt = await acl.burnPermissionManager(kernelAddr, MOCK_ROLE, { from: granted })
+                    assertEvent(receipt, 'ChangePermissionManager')
+
```

Comment:
Missing an assertion that the permission manager is actually `BURN_ENTITY`.

Reply:
@bingen Thoughts on using `acl.getPermissionManager()` to check in both the `'burns existing permission'` and `'burns non-existing permission'` instead of the event?